src/west/app/project.py

```diff
@@ -736,10 +736,19 @@ def print_rev_info(self, project):
             return
 
         def rev_info(rev):
-            return project.git(
-                ['log', '-1', '--color=never', '--pretty=%h %s', rev],
+            title = project.git(
+                ['log', '-1', '--color=never', '--pretty=%h%d %s',
+                 '--decorate-refs-exclude=refs/heads/manifest-rev',
```

Comment:
Did you check that this doesn't introduce a requirement for a more recent version of git?

Reply:
No I hadn't, my bad. These options were introduced in git commit v2.16.0-rc0~64^2 https://github.com/git/git/commit/65516f586b69307f977

Git 2.16.0 was released in January 2018, 5+ years ago.

I couldn't find what is the minimum git version required by west, where is it?

Reply:
That's fine then; we have a `        if self.git_version_info < (2, 22):` check up on line 659.